apps/src/StudioApp.js

```diff
@@ -3000,6 +3004,26 @@ StudioApp.prototype.showRateLimitAlert = function () {
   });
 };
 
+/** @return Promise */
+StudioApp.prototype.loadLibraries = function (helperLibraryNames = []) {
+  if (!this.libraryPreload_) {
+    this.libraryPreload_ = Promise.all(helperLibraryNames.map(this.loadLibrary_.bind(this)));
+  }
+  return this.libraryPreload_;
+};
+
+/** @return Promise */
+StudioApp.prototype.loadLibrary_ = function (name) {
+  if (this.libraries[name]) {
+    return Promise.resolve();
+  }
+
+  return (async () => {
+    const response = await fetch('/libraries/' + name);
+    this.libraries[name] = await response.text();
+  })();
+};
```

Comment:
Note: Instead of the inner async function, you could make the outer function async since it's already returning a Promise.
```js
StudioApp.prototype.loadLibrary_ = async function (name) {
  if (this.libraries[name]) {
    return;
  }

  const response = await fetch('/libraries/' + name);
  this.libraries[name] = await response.text();
};
```